pkg/server/plugin/nodeattestor/gcp/iit_test.go

```diff
@@ -44,6 +46,20 @@ H9A4cjS4cp4W4nBZhb+08/IYtDfYdirhAjAtl8LMtJE045GWlwld+xZ5UwKKSVoQ
 Qj/AwRxXdH++5ycGijkoil4UNzyUtGqPIJkCMQC5g9ola8ekWqKPVxWvK+jOQO3E
 f9w7MoPJkmQnbtOHWXnDzKkvlDJNmTFyB6RwkQECMQDp+GR2I305amG9isTzm7UU
 8pJxbXLymDwR4A7x5vwH6x2gLBgpat21QAR14W4dYEg=
+-----END RSA PRIVATE KEY-----`))
+
+	// Alternative key
+	alternativeKeyPEM, _ = pemutil.ParseRSAPrivateKey([]byte(`-----BEGIN RSA PRIVATE KEY-----
```

Comment:
If we're going to add another key, we might as well use the testkey package for both of these keys while we're at it (i.e. `testkey.MustRSA2048`).